### PR TITLE
docs: Update readme

### PR DESCRIPTION
**Pull request by facundomedica**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,149 @@
...
+## Installation
+
+To install the `peggo` binary:
+
+```shell
+$ make install
+```
+
+## How to run
+### Setup
```
Comment (alexanderbez) [inline README.md:38]:
```suggestion
## How to run

### Setup
```

File: README.md [hunk 2]
```diff
@@ -1,5 +1,149 @@
...
+  --cosmos-from=...
+```
+
+### Send a transfer from Umee to Ethereum
+
+This is done using the command `umeed tx peggy send-to-eth`, use the `--help`
+flag for more information.
+
+If the coin doesn't have a corresponding ERC20 equivalent on the Ethereum
+network, the transaction will fail. This is only required for Umee originated
```
Comment (alexanderbez) [inline README.md:78]: Any cosmos native asset, including IBC tokens ;-)

```suggestion
network, the transaction will fail. This is only required for Cosmos originated
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,149 @@
...
+ **Request batch**: Peggo will check for new transactions in the Outgoing TX Pool
+ and if the transactions' fees are greater than the set minimum batch fee, it
+ will send a message to Umee requesting a new batch.
+
+ **Batches**: Peggo queries Umee for any batches ready to be relayed and relays
+ them over to Ethereum using the `submitBatch` function on the Peggy contract.
+
+### Transfers from Umee to Ethereum
+
+WIP (describe in simple words how a transfer from Umee to Ethereum works)
```
Comment (alexanderbez) [inline README.md:145]: Don't we describe this above?

File: README.md [hunk 4]
```diff
@@ -1,5 +1,149 @@
...
+
+### Transfers from Ethereum to Umee
+
+WIP
```
Comment (alexanderbez) [inline README.md:149]: Ditto

File: README.md (hunk 2 above)
Comment (alexanderbez) [reply README.md:78]: Also, should we mention the helpful command `peggo bridge deploy-erc20`?

Comment (facundomedica) [reply README.md]: Got it 👌

File: README.md (hunk 3 above)
Comment (facundomedica) [reply README.md:146]: I wanted to get a bit more technical here, like describe in more details how the loops work and that stuff

Comment (facundomedica) [reply README.md:147]: I'll remove it for now so it looks cleaner, and then I'll circle back to this

**Review by alexanderbez — APPROVED** [review]

**Pull request merged**